bin/development

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env ruby
 
-require File.join(File.dirname(__FILE__), '..', 'lib', 'parity')
+$LOAD_PATH.unshift File.expand_path(File.join("..", "..", "lib"), __FILE__)
+require "parity"
```

Comment:
If someone has parity added as a Gem in their app's manifest, would this break their path? (I don't think so, but asking because I'm concerned.)